### ISSUE TITLE
Front : Página-Habitaciones

### ISSUE THREAD

**Issue by KassWTF** [we-the-force/h1800-ws#61]
Al añadir más cuartos el precio se actualiza tiempo después, e incluso hay casos en donde nunca se actualiza.

**Link:** http://hacienda1800.com/test/index.php?id_product=20&controller=product&id_lang=1

![Captura de pantalla 2019-10-14 a la(s) 10.35.23.png](https://images.zenhubusercontent.com/5d6fe1b5e370a20001310cd3/e735724b-b609-402f-9a70-bfb0bff782dc)

**Comment by gamanox** [comment]
éste es un jale de lo que vimos de los precios ayer, no? @KassWTF

**Comment by KassWTF** [comment]
No, es sobre la carga de los datos. Ahorita el sitio está muy lento y creo que es lo que influye. @gamanox
